src/packages/debug/package.json

```diff
@@ -36,19 +36,5 @@
   ],
   "dependencies": {
     "debug": "^4.1.1"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
```

Comment:
If this was removed then we don't need the dev dependencies associated?

Reply:
this needs to be checked in all pkg

Reply:
This will come back, just wip